.github/workflows/standard_spa_pr_NPM.yml

```diff
@@ -321,7 +321,7 @@ jobs:
       - id: outputStep
         # Use jq to extract the accountId and user sessions from the json file in a raw form
         run: |
-          echo "artemisAccount=$(jq -r .[0].id < artemis-run.json)" >> $GITHUB_OUTPUT
+          echo "artemisAccountName=$(jq -r .[0].metadata.accountName < artemis-run.json)" >> $GITHUB_OUTPUT
```

Comment:
this is the bug/fix, all the other changes are renames `artemisAccount` -> `artemisAccountName`